Source/PeleLM.cpp

```diff
@@ -5768,12 +5768,10 @@ PeleLM::advance_chemistry (MultiFab&       mf_old,
         time_chem = 0;
 #else
         int reactor_type = 2;
-        tmp_fctCn[0] = react(tmp_vect, tmp_src_vect,
-                      tmp_vect_energy, tmp_src_vect_energy,
-                      dt_incr, time_chem,
-                      reactor_type, ncells, 
-                      amrex::Gpu::gpuStream());
+        int tmp_fctCn = react(bx, rhoY, frc_rhoY, temp, rhoH, frc_rhoH, fcl, mask, 
+                              dt_incr, time_chem, reactor_type, amrex::Gpu::gpuStream());
```

Comment:
Why is ``reactor_type`` passed here and also in the init function?

Reply:
You need the reactor_type because on tjhe GPu you create a new cvode instance each time you call react(). You actually don't need the init function on  the GPu. And that's actually why it's not called react_init but react_info. It simply prints info for you at the begining of the run so you can check that you setup everything correctly